web-client/src/presenter/computeds/blockedCasesReportHelper.js

```diff
@@ -32,12 +33,28 @@ export const blockedCasesReportHelper = (get, applicationContext) => {
     blockedCasesFormatted = blockedCases
       .sort(applicationContext.getUtilities().compareCasesByDocketNumber)
       .map(blockedCase => {
+        const inConsolidatedGroup = !!blockedCase.leadDocketNumber;
+        const inLeadCase =
+          inConsolidatedGroup &&
+          blockedCase.leadDocketNumber === blockedCase.docketNumber;
+
+        if (inConsolidatedGroup) {
+          if (inLeadCase) {
+            consolidatedIconTooltipText = 'Lead case';
+          } else {
+            consolidatedIconTooltipText = 'Consolidated case';
+          }
+        }
```

Comment:
This seems to be a little more complicated than it needs to be since `inLeadCase` already includes `inConsolidatedGroup` on line 38.
```suggestion
        consolidatedIconTooltipText = inConsolidatedGroup && 'Consolidated case';
        consolidatedIconTooltipText = inLeadCase && 'Lead case';
```

Honestly I still find this syntax a bit esoteric, but it keeps the nested if away. Alternatively, you could leave out line 38 and leave this code as-is or even make a slight change to the code I've proposed and achieve the same thing.

Reply:
Actually, no, that suggestion won't work, because the second one could set to false. It would need to be a ternary like:
```suggestion
        if (inConsolidatedGroup) {
          if (inLeadCase) {
            consolidatedIconTooltipText = 'Lead case';
          } else {
            consolidatedIconTooltipText = 'Consolidated case';
          }
        }
        consolidatedIconTooltipText = inLeadCase ? 'Lead case' : inConsolidatedGroup ? 'Consolidated case' : null
```

Reply:
I am not sure how this makes it more readable.  Personal opinion, a nested ternary operator is harder to read and if we are going to leave in the nested if, why then the need to nested ternary operator.

Reply:
I definitely see the sentiment to make this clearer, but it's refreshing my memory that originally with the first helper, we went through this same thought process. Nested if => Ternary => nested if for readability 🤷